libs/vkd3d/device.c

```diff
@@ -4848,6 +4914,19 @@ static void d3d12_device_caps_override(struct d3d12_device *device)
         caps->options.ConservativeRasterizationTier = max(caps->options.ConservativeRasterizationTier, D3D12_CONSERVATIVE_RASTERIZATION_TIER_1);
     }
 
+    if (fl_override >= D3D_FEATURE_LEVEL_12_2)
+    {
+        caps->options5.RaytracingTier = max(caps->options5.RaytracingTier, D3D12_RAYTRACING_TIER_1_1);
+        caps->options6.VariableShadingRateTier = max(caps->options6.VariableShadingRateTier, D3D12_VARIABLE_SHADING_RATE_TIER_1);
+        caps->options.ResourceBindingTier = max(caps->options.ResourceBindingTier, D3D12_RESOURCE_BINDING_TIER_3);
+        caps->options.TiledResourcesTier = max(caps->options.TiledResourcesTier, D3D12_TILED_RESOURCES_TIER_3);
+        caps->options.ConservativeRasterizationTier = max(caps->options.ConservativeRasterizationTier, D3D12_CONSERVATIVE_RASTERIZATION_TIER_3);
+        caps->max_shader_model = max(caps->max_shader_model, D3D_SHADER_MODEL_6_5);
+        /* FIXME: 40-bit virtual address space, not sure if that's a query? */
+        /* FIXME: Mesh shader? Doesn't seem to be a caps bit for it. */
+        /* FIXME: Sampler feedback? */
```

Comment:
There are cap bits in `D3D12_FEATURE_DATA_D3D12_OPTIONS7`, which would need to be added to `d3d12_caps` and handled in CheckFeatureSupport.

Reply:
Added it.